sdk/objectanchors/Azure.MixedReality.ObjectAnchors/README.md

```diff
@@ -0,0 +1,144 @@
+# Azure Object Anchors client library for .NET
+
+Azure Object Anchors enables an application to detect an object in the physical world using a 3D model and estimate its 6-DoF pose. This package supports the conversion of an existing 3D asset into a form that can be used by the Object Anchors runtime to detect physical objects.
```

Comment:
@craigktreasure 
Adopted suggested language
RE: "I might suggest stealing some of the language from the docs.", and the suggested change